src/NUnitTestAdapter/TfsTestFilter.cs

```diff
@@ -90,19 +96,10 @@ public TfsTestFilter(IRunContext runContext)
 
 
         private ITestCaseFilterExpression testCaseFilterExpression;
-        public ITestCaseFilterExpression TfsTestCaseFilterExpression
-        {
-            get
-            {
-                return testCaseFilterExpression ??
-                       (testCaseFilterExpression = runContext.GetTestCaseFilter(SupportedProperties, PropertyProvider));
-            }
-        }
+        public ITestCaseFilterExpression TfsTestCaseFilterExpression => testCaseFilterExpression ??
+                                                                        (testCaseFilterExpression = runContext.GetTestCaseFilter(SupportedProperties, PropertyProvider));
 
-        public bool IsEmpty
-        {
-            get { return TfsTestCaseFilterExpression == null || TfsTestCaseFilterExpression.TestCaseFilterValue == string.Empty; }
-        }
+        public bool IsEmpty => TfsTestCaseFilterExpression == null || TfsTestCaseFilterExpression.TestCaseFilterValue == string.Empty;
```

Comment:
Some of these lines get pretty long...

One option for long getters would also be
```cs
{
    get => TfsTestCaseFilterExpression == null || TfsTestCaseFilterExpression.TestCaseFilterValue == string.Empty;
}
```